pipeline/metadata/6.0.yaml

```diff
@@ -499,6 +499,16 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - rbd
+        - name: "RGW bucket notifications testing on Multisite"
```

Comment:
@hmaheswa, could you please add this suite under stage-5, stage-6 already have 5 suites